block-streamer/src/graphql/client.rs

```diff
@@ -0,0 +1,126 @@
+use crate::graphql::queries::get_bitmaps_exact::{get_bitmaps_exact, GetBitmapsExact};
+use crate::graphql::queries::get_bitmaps_wildcard::{get_bitmaps_wildcard, GetBitmapsWildcard};
+use ::reqwest;
+use graphql_client::{GraphQLQuery, Response};
+use std::error::Error;
+
+// TODO: Use Dataplatform account
+const HASURA_ACCOUNT: &str = "darunrs_near";
+
+pub struct GraphqlClient {
+    client: reqwest::Client,
+    graphql_endpoint: String,
+}
+
+#[cfg_attr(test, mockall::automock)]
+impl GraphqlClient {
+    pub fn new(graphql_endpoint: String) -> Self {
+        Self {
+            client: reqwest::Client::new(),
+            graphql_endpoint,
+        }
+    }
+
+    pub async fn get_bitmaps_exact(
+        &self,
+        receiver_ids: Vec<String>,
+        block_date: String,
+        limit: i64,
+        offset: i64,
+    ) -> Result<
+        Vec<get_bitmaps_exact::GetBitmapsExactDarunrsNearBitmapV5ActionsIndex>,
+        Box<dyn Error>,
+    > {
```

Comment:
```suggestion
    ) -> anyhow::Result<Vec<get_bitmaps_exact::GetBitmapsExactDarunrsNearBitmapV5ActionsIndex>> {
```

`anyhow` is essentially the same, but is easier to work with